formats/detailed_guide/publisher/links.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "lead_organisations"
+  ],
+  "properties": {
+    "related_guides": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "alternative_format_provider_organisation": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "related_mainstream": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "additional_related_mainstream": {
```

Comment:
The links aren't ordered, so the details field needs to contain an array of the content ids in order. The frontend will then map that ordering to the actual expanded links content. There's no concept of "first" from the links hash itself.